src/OAuth2Form.tsx

```diff
@@ -1,403 +1,439 @@
+          })
+          .catch(_err => {
+            // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+            onError(_err)
           })
-          .catch((_err) => {
-            onError(_err);
-          });
       }
-    };
+    }
 
-    getOauthClientInfo();
-  }, []);
+    getOauthClientInfo()
+  }, [])
 
   useEffect(() => {
     if (redirectURL) {
-      window.location.replace(redirectURL);
+      window.location.replace(redirectURL)
     }
-  }, [redirectURL]);
+  }, [redirectURL])
 
   const loadingSpinner = (
-    <div style={{ textAlign: "center" }}>
-      <div style={{ marginTop: "50px" }}>
+    <div style={{ textAlign: 'center' }}>
+      <div style={{ marginTop: '50px' }}>
         {redirectURL && oauthClientInfo && (
           <p>Waiting for {oauthClientInfo.client_name}...</p>
         )}
         <span
           style={{
-            marginLeft: "10px",
-            backgroundSize: "40px 40px",
-            width: "40px",
-            height: "40px",
+            marginLeft: '10px',
+            backgroundSize: '40px 40px',
+            width: '40px',
+            height: '40px',
           }}
-          className={"spinner"}
+          className={'spinner'}
         />
       </div>
     </div>
-  );
+  )
 
   return (
     <div>
       {!error && oauthClientInfo && !oauthClientInfo.verified && (
-        <>
-          <div className="BlueBackground">
-            <div className="ComponentWrapper">
-              <WarningTwoTone
-                className="text-danger"
-                style={{
-                  marginLeft: "5px",
-                  marginBottom: "15px",
-                  fontSize: "40px",
-                }}
-              />
-              <h3>This app isn&apos;t verified</h3>
-              <p>This app has not been verified by Sage Bionetworks yet.</p>
-              <div className="text-align-right margin-top-40">
-                <button onClick={onGoBack} className="btn btn-primary">
-                  Back to Safety
-                </button>
-              </div>
+        <StyledOuterContainer>
+          <StyledInnerContainer>
+            <WarningTwoTone
+              className="text-danger"
+              style={{
+                marginLeft: '5px',
+                marginBottom: '15px',
+                fontSize: '40px',
+              }}
+            />
+            <h3>This app isn&apos;t verified</h3>
+            <p>This app has not been verified by Sage Bionetworks yet.</p>
+            <div className="text-align-right margin-top-40">
+              <button onClick={onGoBack} className="btn btn-primary">
+                Back to Safety
+              </button>
             </div>
-          </div>
-        </>
+          </StyledInnerContainer>
+        </StyledOuterContainer>
       )}
       {!redirectURL &&
         !error &&
+        isPreviousAuthCheckComplete &&
         accessToken &&
+        !isConsenting &&
```

Comment:
Show the consent form if we checked for previous auth (skip) and we are not in the middle of consenting